src/gui/GuiApplication.cpp

```diff
@@ -73,8 +79,11 @@ GuiApplication* GuiApplication::instance()
 
 
 
-GuiApplication::GuiApplication()
+GuiApplication::GuiApplication(int* sigintFd)
```

Comment:
If there's a better way to pass this in without modifying the constructor, please let me know.   I expected static functions to handle this, but they behave different in C++ versus other languages.  I tried `extern`, but that wanted a member variable.  I'd much rather not have a separate constructor for Windows.